pipeline-model-api/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/ast/ModelASTAxis.java

```diff
@@ -19,7 +19,7 @@
 public class ModelASTAxis extends ModelASTElement {
 
     private ModelASTKey name;
-    private List<ModelASTValue> values = new ArrayList<>();
+    private List<ModelASTValue> values;
```

Comment:
So this change is causing test failures - I far prefer always initing collections to empty rather than null, to avoid this kind of problem (and to avoid needing to bake in initing to empty in multiple places instead). What's the reasoning for this change?

Reply:
I was attempting to be able to know when `values` was missing vs when it has no args passed, but It is definitely more trouble than it is worth.